src/node/rpc/memberfrontend.h

```diff
@@ -713,8 +719,13 @@ namespace ccf
               fmt::format("No ACK record exists for caller {}", caller_id));
           }
 
-          auto root = sig_view->get(0)->root;
-          ma->state_digest = std::vector<uint8_t>(root.h.begin(), root.h.end());
+          auto s = sig_view->get(0);
+          if (s)
```

Comment:
If `!s`, I guess there's no point `put`ting the same `ma` value in the `ma_view`?

Reply:
true :)